bin/github-changelog-generator

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env php
+<?php
+
+require_once '../vendor/autoload.php';
+
+// define CLI arguments
+$cli = new Commando\Command();
```

Comment:
Just asking what would be the downside to just getting the args insted of using this dependencie?